config/cmake/H5pubconf.h.in

```diff
@@ -480,12 +477,6 @@
 /* Define to 1 if you have the `_scrsize' function. */
 #cmakedefine H5_HAVE__SCRSIZE @H5_HAVE__SCRSIZE@
 
-/* Define if the compiler understands __inline */
-#cmakedefine H5_HAVE___INLINE @H5_HAVE___INLINE@
-
-/* Define if the compiler understands __inline__ */
-#cmakedefine H5_HAVE___INLINE__ @H5_HAVE___INLINE__@
-
 /* Define if the library will ignore file locks when disabled */                 
```

Comment:
I would leave these symbols here and pull them in the CMake PR. Having them here makes it difficult to propagate the inline code changes.

Reply:
I'm a bit confused by this.  You want me to deliberately make a PR (this one) that only half-removes the old inline workarounds, and leaves that `#cmakedefine` statement as dead code?

Reply:
Yes. Exactly that. We can change the code but leave the configure/CMake bits in older branches.